manifests/auth.pp

```diff
@@ -69,6 +70,7 @@
   Integer[0]                     $cracklib_retry            = $pam::cracklib_retry,
   Integer[0]                     $deny                      = $pam::deny,
   Boolean                        $faillock                  = $pam::faillock,
+  Optional[Stdlib::Absolutepath] $faillock_log_file         = undef,
```

Comment:
Would recommend making this `faillock_lock_dir` since it's a directory target.

Also, if this gets moved from the default, the proper permissions and SELinux context are going to need to be set on it or it won't work.

```
ll -dZ /var/run/faillock/
drwxr-xr-x. 2 root root system_u:object_r:faillog_t:s0 40 Mar 14 12:17 /var/run/faillock/
```

Reply:
I believe the issues above were addressed in the latest commit, let me know if anything else looks amiss